machine_learning/lstm/lstm_prediction.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
+import numpy as np
+from sklearn.preprocessing import MinMaxScaler
+from keras.models import Sequential
+from keras.layers import LSTM,Dense
+
+def clean_data(data_file):
+	'''
+	This function will read and clean data
+	input: CSV file containing data
+	output: Data which will be useful for training
+	'''
+	df = pd.read_csv(data_file, header=None)
+	print(df.head())
+	len_data = df.shape[:1][0]
+	actual_data = df.iloc[:,1:2]
+	actual_data = actual_data.values.reshape(len_data,1)
+	scl = MinMaxScaler()
```

Comment:
`scl` --> `min_max_scaler` (don't make your reader guess) or just do `MinMaxScaler().fit_transform(actual_data)` if it is a one-time use.